zen_queries/utils.py

```diff
@@ -1,3 +1,2 @@
 def fetch(queryset):
     queryset._fetch_all()
-    return queryset
```

Comment:
This is a massive breaking change and will break a pattern that I use all the time:

```
widgets = fetch(Widget.objects.all())
```